modules/tinymce/src/themes/silver/main/ts/modes/Iframe.ts

```diff
@@ -158,7 +158,8 @@ const render = (editor: Editor, uiComponents: RenderUiComponents, rawUiConfig: R
     },
     disable: () => {
       ReadOnly.broadcastReadonly(uiComponents, true);
```

Comment:
I definitely could be wrong here, as I don't understand much of what is going on, but should we be using the Disabling api here also?

Reply:
This does use the disabling APIs, just not directly, as everything is hooked up to the event since you can't call disabling on every component from here. So you fire the event, the components that support readonly listen for the event and then call `Disabling.set(comp, state)` from the receiving config.

Reply:
Awesome